client/src/components/profile/Profile.css

```diff
@@ -0,0 +1,142 @@
+*,
+ ::after,
+ ::before {
+    box-sizing: border-box;
+}
+
+.card-profile {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+    max-width: 300px;
+    margin: 10px;
+    background: #F2F2F2;
+    font-size: .8em;
+    text-align: center;
+    padding: 10px
+}
+
+.title-name {
+    color: #0D7E6A;
+    font-size: 18px;
+    line-height: 35px;
+}
+
+.title-about {
+    color: #0D7E6A;
+    font-size: 18px;
+    line-height: 35px;
+    border-bottom: 2px solid #FCA73D;
+}
+
+.title-help {
+    color: #0D7E6A;
+    font-size: 16px;
+    line-height: 20px;
+    margin-top: 60px;
+}
+
+.help {
+    color: #CC0A0A;
+}
+
+.icon {
+    text-align: center;
+    color: #D8D8D8;
+    font-size: 0.9em;
+    margin-bottom: 2px;
+}
+
+.bio-icon {
+    /*
+    margin-bottom: 2px;
+    left: 545px;
+    top: 498px;
+    border-radius: 4px;
+    display: inline-block;
+    float: left; */
+    background: #F2F2F2;
+    font-size: 1.5em;
+    color: #4f4f4f;
+    width: 45px;
+    height: 45px;
+    text-align: center;
+}
+
+.bio-h2 {
+    font-size: 1em;
+    font-weight: 400;
+    font-style: italic;
+    color: #BDBDBD;
+    /* display: inline-block; */
+}
+
+.card-help {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+    max-width: 400px;
+    margin: 10px;
+    background: #F2F2F2;
+    font-size: .8em;
+    padding: 10px;
+}
+
+.card-comment {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+    max-width: 400px;
+    margin: 10px;
+    background: #F2F2F2;
+    font-size: .8em;
+    padding: 10px;
+}
+
+.help-comment {
+    display: flex;
+    flex-direction: column;
+    border-color: white;
+}
+
+.form-comment input {
+    width: 100%;
+    height: 40px;
+    font-size: 1em;
+}
+
+.form-comment input[type=text] {
+    border: 0 solid #F2F2F2;
+}
+
+.rowC {
+    display: flex;
+    flex-direction: row;
+    margin-left: 5%;
+    margin-top: 5%;
```

Comment:
@LujiAnna I checked your style , I made a small edit in the margins on the profile page, can you change this;
`margin-left: 8%;
 margin-right: 8%;` instead of line 109-110
 and try please

Reply:
And after this changes, can you add `w-50` class name in `bio.jsx line 33`

Reply:
edited..